app/models/transliterated_appointment_summary.rb

```diff
@@ -0,0 +1,9 @@
+class TransliteratedAppointmentSummary < SimpleDelegator
+  %i(first_name last_name guider_name
+     address_line_1 address_line_2 address_line_3
+     town county postcode country).each do |method_name|
+    define_method(method_name) do
+      I18n.transliterate(super())
```

Comment:
Unfortunately that doesn't work, you get the following error `implicit argument passing of super from method defined by define_method() is not supported. Specify all arguments explicitly.`